mlflow/server/js/src/experiment-tracking/components/ExperimentPage.js

```diff
@@ -29,6 +29,10 @@ export const MODEL_VERSION_FILTER = {
   ALL_RUNS: 'All Runs',
 };
 
+export const DEFAULT_ORDER_BY_KEY = 'attributes.start_time';
```

Comment:
Can we use [this prop](https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/experiment-tracking/components/ExperimentViewUtil.js#L212) here so we have the constant defined in one place?